src/src/rx-serial/SerialHoTT_TLM.h

```diff
@@ -0,0 +1,343 @@
+                                              // 0    mAh
+                                              // 1    Battery 1
+                                              // 2    Battery 2
+                                              // 3    Temperature 1
+                                              // 4    Temperature 2
+                                              // 5    Altitude
+                                              // 6    Current
+                                              // 7    Main power voltage
+    uint8_t inverse2 = 0;                     //  5 alarm bitmask. Value is displayed inverted
+                                              // Bit#  Alarm Field
+                                              // 0    m/s
+                                              // 1    m/3s
+                                              // 2    Altitude (duplicate?)
+                                              // 3    m/s (duplicate?)
+                                              // 4    m/3s (duplicate?)
+                                              // 5    unknown/unused
+                                              // 6    unknown/unused
+                                              // 7    "ON" sign/text msg active
+    uint8_t cellL[7] = {0};                   // 06 cell 1 voltage lower value. 0.02V steps, 124=2.48V
+    uint8_t cellH[7] = {0};                   // 13 cell 1 voltage high value. 0.02V steps, 124=2.48V
+    uint16_t battVoltage1 = 0;                // 20 battery 1 voltage lower value in 100mv steps, 50=5V. optionally cell8_L value 0.02V steps
+    uint16_t battVoltage2 = 0;                // 22 battery 2 voltage lower value in 100mv steps, 50=5V. optionally cell8_H value. 0.02V steps
+    uint8_t temp1 = 20;                       // 24 Temperature sensor 1. 20=0�, 46=26� - offset of 20.
+    uint8_t temp2 = 20;                       // 25 temperature sensor 2
+    uint16_t altitude = 500;                  // 26 Altitude lower value. unit: meters. Value of 500 = 0m
+    uint16_t current = 0;                     // 28 Current in 0.1 steps
+    uint16_t mainVoltage = 0;                 // 30 Main power voltage (drive) in 0.1V steps
+    uint16_t capacity = 0;                    // 32 used battery capacity in 10mAh steps
+    uint16_t mPerSec = 30000;                 // 34 climb rate in 0.01m/s. Value of 30000 = 0.00 m/s
+    uint8_t mPer3sec = 120;                   // 36 climbrate in m/3sec. Value of 120 = 0m/3sec
+    uint16_t rpm = 0;                         // 37 RPM. Steps: 10 U/min
+    uint8_t electricMin = 0;                  // 39 Electric minutes. Time does start, when motor current is > 3 A
+    uint8_t electricsec = 0;                  // 40 Electric seconds.
+    uint16_t speed = 0;                       // 41 speed in km/h. Steps 1km/h
+    uint8_t endByte = END_FRAME;              // 43 0x7D
+    uint8_t Crc;                              // 44 CRC
```

Comment:
You missed one 😄 
I'm pretty happy otherwise. Once this one is fixed I'll give it the 👍

Reply:
done - waiting for my atta boy

Reply:
Atta boy 👍